### ISSUE TITLE
[LibOS] Add support for Musl libc

### ISSUE THREAD

**Issue by dimakuv** [oscarlab/graphene#746]
Currently Graphene supports only Glibc as the standard C library. Add support for Musl libc.

This will be beneficial for IoT/embedded users.

**Comment by mkow** [comment]
Blocked by #2205.

**Comment by dimakuv** [comment]
The PR is there (#2269) but it is currently blocked on Meson rewrite and proper installation/packaging of Graphene.

**Comment by mkow** [comment]
As discussed on the call, this should have lower priority.
